### PR TITLE
feat: add mem object group & apisix model

### PR DESCRIPTION
**Pull request by gxthrj**

### CODE REVIEW

File: pkg/dp/apisix/route.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	"encoding/json"
+	"fmt"
+	"github.com/apache/apisix-control-plane/pkg/conf"
+	"github.com/apache/apisix-control-plane/pkg/utils"
+)
+
+func (route *Route) Add() (*RouteResponse, error) {
+	baseUrl := conf.BaseUrl
+	url := fmt.Sprintf("%s/routes", baseUrl)
+	if b, err := json.Marshal(route); err != nil {
```
Comment (membphis) [inline pkg/dp/apisix/route.go:30]: Bad code style, here is a new one.

```golang
if ...{
    return ...
}

if ... {
    return ....
}

return ...
```
```

File: pkg/dp/apisix/upstream.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	"encoding/json"
+	"fmt"
+	"github.com/apache/apisix-control-plane/pkg/utils"
+	"github.com/gxthrj/seven/conf"
+)
+
+func (upstream *Upstream) Add() (*UpstreamResponse, error) {
+	baseUrl := conf.BaseUrl
+	url := fmt.Sprintf("%s/upstreams", baseUrl)
+	if b, err := json.Marshal(upstream); err != nil {
```
Comment (membphis) [inline pkg/dp/apisix/upstream.go:30]: ditto

**Review by membphis — COMMENTED** [review]
missing test case?

File: pkg/dp/apisix/route.go (hunk 1 above)
Comment (gxthrj) [reply pkg/dp/apisix/route.go:30]: This is some style of golang, many gophers like this style

File: pkg/dp/apisix/upstream.go (hunk 2 above)
Comment (gxthrj) [reply pkg/dp/apisix/upstream.go:30]: This is some style of golang, many gophers like this style

**Comment by gxthrj** [comment]
> missing test case?

will Add later

**Pull request merged**